### ISSUE TITLE
[REQUEST] XOR Encryption

### ISSUE THREAD

**Issue by mastercho** [openbullet/openbullet#571]
**Is your feature request related to a problem? Please describe.**
Some websites would encrypt passwords with XOR and would be good to be added in OB

**Describe the solution you'd like**
`        private static string XorEncrypt(string xorToEncrypt)
        {
            string encryptedString = "";
            foreach (byte chara in Encoding.Default.GetBytes(xorToEncrypt))
            {
                encryptedString += Convert.ToInt32(chara ^ 5).ToString("x");
            }
            return encryptedString;
        }
`

**Comment by openbullet** [comment]
You're basically xoring every character's ascii code with the number 5. Why? Shouldn't there be 2 arguments of equal length in the function?

**Comment by mastercho** [comment]
Ugh i just tried to convert PHP code which i was using from https://github.com/mgp25/TikTok/blob/master/src/Signatures.php into C# and was working great

**Comment by openbullet** [comment]
Yes, for that case specifically, but it's not good to add a function with hardcoded parameters in OB so it needs to be done properly, not like you posted it.
